src/commands/diag.js

```diff
@@ -28,12 +28,16 @@ async function diag () {
     Logger.println('Linux         ' + colors.green(linuxInfos));
   }
 
+  Logger.println('Shell         ' + colors.green(process.env.SHELL));
+
   const isPackaged = (process.pkg != null);
   Logger.println('Packaged      ' + colors.green(isPackaged));
   Logger.println('Exec path     ' + colors.green(process.execPath));
+  Logger.println('Config file   ' + colors.green(conf.CONFIGURATION_FILE));
 
   Logger.println('User id       ' + colors.green(userId || 'Not connected'));
```

Comment:
In case the user is not connected, maybe we can hide the information below (the oAuthToken and the auth source)?

Reply:
Agreed! :+1: